src/components/AddTask.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import { ReactComponent as DeleteIcon } from 'assets/delete.svg';
+import { useDispatch } from 'react-redux';
+import { addTodo, setIsAddingTask } from 'redux/actions/TodoAction';
+import { ENTER } from 'utils/constants';
+import { validate } from 'utils/helpers/index';
+
+function AddTask() {
+  const [error, setError] = useState(null);
+  const [taskDetails, setTaskDetails] = useState('');
+  const dispatch = useDispatch();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const validateDetails = validate(taskDetails);
+    if (validateDetails.status === 'error') {
```

Comment:
Could we use const for strings ?
```suggestion
    if (validateDetails.status === ERROR) {
```

Reply:
Constant added for this. thanks